_data/download-descriptions.yml

```diff
@@ -200,6 +200,8 @@ linux-ubuntu-xenial-i386-dbg.deb:
   description: Debug Symbols for Linux Ubuntu Xenial 16.04 (i386, 32 bit)
 linux-ubuntu-xenial-i386.deb:
   description: Linux Ubuntu Xenial 16.04 (i386, 32 bit)
+macos-universal.dmg:
+  description: macOS 10.9+ (Universal)
```

Comment:
Is mentioning "Universal" clear enough for people? I kinda miss the mention it is Intel and Arm/Silicon, as universal could be all kinds of combinations. I don't know what other software does, but if I bought an M1, not sure that Universal gave me the idea that it would run native, as I would be thinking: no way they ported their software this quick :P

I don't know the MacOS world enough, just asking the question :D

Reply:
The binary shows up as "Universal", "Intel" or "Apple Silicon" in the Mac's Get Info dialog, so this is consistent with Apple's usage. I don't think it would cause confusion.

The old (PPC/Intel) Universal binaries haven't really been around for a decade or so.